src/lucky/welcome_page.cr

```diff
@@ -41,12 +38,87 @@ class Lucky::WelcomePage
     raw %(<img src="#{lucky_logo_data_uri}" class="lucky-logo">)
   end
 
-  def lucky_welcome_tagline
-    h1 "Today is a great day to build an app", class: "hero-tagline"
+  def render_buttons
+    div class: "container__buttons" do
+      a "Check out the guides",
+        href: "https://luckyframework.org/guides",
+        class: "btn btn--blue"
+      render_auth_button
+    end
+  end
+
+  def render_help
+    h1 "Not sure where to start? Here are some ideas:", class: "headline"
+
+    table class: "help-table" do
+      tbody do
+        tr do
+          td "Change the page this URL renders", class: "left-column"
```

Comment:
What do you think about

> Edit this page in: `src/actions/home/index.cr`

Reply:
Hmm, you're not actually changing the page though, your changing the page it renders or the action it redirects to. Open to other ideas though. I'm not super sold on what's there!

Reply:
Hmm action is sort of a tough concept to get across. Maybe

> Change the backend of this page

Reply:
That seems a bit misleading to me. Almost like you'll change the whole framework :P
 
Maybe `Change the what this page renders`? So you don't really need to start off knowing about actions and all that?

Reply:
Hah! `Pull requests welcome` 😉 

I like what you put but

```diff
-Change the what this page renders
+Change what this page renders
```

Reply:
Nah...I like the typo better 😎